mungegithub/submit-queue/deployment/contrib/configmap.yaml

```diff
@@ -7,7 +7,7 @@ data:
   config.http-cache-dir: /cache/httpcache
   config.organization: kubernetes
   config.project: contrib
-  config.pr-mungers: blunderbuss,size
```

Comment:
The currently running deployment has size removed as well. We need the labels to be created using the check-labels munger before turning on size, or it complains about being unable to find various size labels.